programs/mpl-core/src/utils.rs

```diff
@@ -438,6 +520,24 @@ pub(crate) fn validate_collection_permissions<'a>(
         }
     };
 
+    if let Some(external_plugin_validate_fp) = external_plugin_validate_fp {
+        match validate_external_plugin_checks(
+            Key::CollectionV1,
+            &external_checks,
+            authority_info,
+            None,
+            new_plugin,
+            None,
+            Some(collection),
+            &resolved_authorities,
+            external_plugin_validate_fp,
+        )? {
+            ValidationResult::Approved => approved = true,
+            ValidationResult::Rejected => rejected = true,
+            ValidationResult::Pass | ValidationResult::ForceApproved => (),
```

Comment:
Ditto on force approve.

Reply:
See other comment, I think this might be the easy way to be "chucking" it.